viennacl/ocl/context.hpp

```diff
@@ -115,6 +115,12 @@ class context
     return devices_[current_device_id_];
   }
 
+  /** @brief Returns the current device index */
+  int current_device_id() const
```

Comment:
Can you please adjust the return type to vcl_size_t to avoid implicit conversions?